src/ru/skypro/Employee.java

```diff
@@ -1,4 +1,87 @@
 package ru.skypro;
 
+import java.util.Objects;
+
 public class Employee {
+    private static int counter = 1;
+    private final int id;
+    private final String firstName;
```

Comment:
хорошим тоном считается визуальное отделение статическим полей от обычных пустой строкой, чтобы их сгруппировать для лучшей читаемости кода
```suggestion
    private static int counter = 1;
    
    private final int id;
    private final String firstName;
```